package.json

```diff
@@ -12,10 +12,8 @@
   "scripts": {
     "prebuild": "rimraf dist lib && npm run build:lib && npm run build:ts",
     "build:ts": "tsc --declaration --declarationMap --emitDeclarationOnly",
-    "build:tailwind": "tailwindcss build .storybook/tailwind.css -o .storybook/output.css",
```

Comment:
this need for dev mode build, no?

Reply:
no, it was being previously but not anymore.